.github/workflows/ci-cd.yaml

```
@@ -7,52 +7,92 @@
+        uses: ./.github/actions/publish-image
+        with:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          dockerImageBaseName: ghcr.io/altinn/altinn-correspondence
+          imageTag: ${{ needs.get-version.outputs.imageTag }}
+
   deploy-test:
     name: Internal test
-    uses: ./.github/workflows/deploy-to-environment.yml
+    runs-on: ubuntu-latest
+    needs: [get-version, publish]
     if: always() && !failure() && !cancelled() 
```

yamllint

[error] trailing spaces

(trailing-spaces)